admin_manual/installation/linux_installation.rst

```diff
@@ -14,6 +14,15 @@ Installing ownCloud Enterprise Subscription
 See :doc:`../enterprise_installation/linux_installation` for instructions on 
 installing ownCloud Enterprise Subscription.
 
+Downgrading Not Supported
```

Comment:
Sorry, a little bit too late but is the installation chapter the right place for this note?